Pano/Views/LoginRegister/LoginRegisterView.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  LoginRegisterView.swift
+//  Pano
+//
+//  Created by Serdar Onur KARADAĞ on 18.07.2023.
+//
+
+import SwiftUI
+
+struct LoginRegisterView: View {
```

Comment:
I would suggest naming this class as something like `AuthorizationView` or `AuthenticationView`.